src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -216,6 +224,294 @@ public static class PublishedContentExtensions
+                {
+                    // Simple types
+                    propertyInfo.SetValue(instance, propertyValue, null);
+                }
+                else
+                {
+                    using (DisposableTimer.DebugDuration(type, string.Format("TypeConverter ({0}, {1})", content.Id, propertyInfo.Name), "Complete"))
+                    {
+                        var convert = propertyValue.TryConvertTo(propertyInfo.PropertyType);
+                        if (convert.Success)
+                        {
+                            propertyInfo.SetValue(instance, convert.Result, null);
+                        }
+                    }
+                }
+            }
+        }
+
+        private static PropertyInfo[] AddToPropertyCache(Type type)
+        {
+            PropertyInfo[] properties;
+            PropertyCache.TryGetValue(type, out properties);
+
+            if (properties == null)
+            {
+                properties = type.GetProperties(BindingFlags.Public | BindingFlags.Instance)
+                        .Where(x => x.CanWrite)
+                        .ToArray();
+
+                PropertyCache.TryAdd(type, properties);
+            }
+
+            return properties;
+        }
+
+        private static object GetTypedArchetype(IPublishedContent content, Type entityType, ArchetypeCMSModel archetype)
+        {
+            var isGenericList = entityType.IsGenericType && (entityType.GetGenericTypeDefinition() == typeof(IList<>) || entityType.GetGenericTypeDefinition() == typeof(List<>));
+
+            Type archetypeListType = null;
+
+            archetypeListType = isGenericList ? entityType.GetGenericArguments().FirstOrDefault() : entityType;
+
+            if (archetypeListType == null)
+            {
+                throw new NullReferenceException(string.Format("The type ({0}) can not be inferred?", entityType.Name));
+            }
+
+            // ML - Build a generic list from the type fuond above
+
+            var constructedListType = typeof(List<>).MakeGenericType(archetypeListType);
+            var list = (IList)Activator.CreateInstance(constructedListType);
+
+            if (archetype != null && (archetype.Fieldsets != null && archetype.Fieldsets.Any()))
+            {
+                if (archetype.Fieldsets != null && archetype.Fieldsets.Any())
+                {
+                    foreach (var fieldset in archetype.Fieldsets)
+                    {
+                        if (fieldset.Properties != null && fieldset.Properties.Any())
+                        {
+                            Type instanceType = null;
+
+                            ArchetypeCache.TryGetValue(fieldset.Alias, out instanceType);
+
+                            if (instanceType == null)
+                            {
+                                instanceType = TypeInferenceExtensions.GetTypeByName(fieldset.Alias, typeof(ArchetypeModel)).FirstOrDefault();
```

Comment:
@leekelleher Yeah, that's exactly what its doing.

Though the code doesn't explicitly use the PropertyValueConverters here, it does fall back to using this in the SetValue method contained within the class, the same way ditto does as its final fallback binding attempt.